tools/rdock/rdock.xml

```diff
@@ -0,0 +1,60 @@
+<tool id="rdock" name="rDock" version="1.0">
+    <description>Binding Mode Prediction in Proteins/RNA</description>
+    <requirements>
+        <requirement type="package" version="2013.1">rDock</requirement>
+    </requirements>
+    <command>
+        <![CDATA[ 
+    	ln -s '$inputmol2' inputmol2.mol2 && cat '$inputprm' | sed 's|RECEPTOR_FILE.*|RECEPTOR_FILE inputmol2.mol2|g' | sed 's|REF_MOL.*|REF_MOL ligand.sd|g' > new.prm && 
+        mkdir -p data/ligands &&
+        cp '$inputsd' data/ligands/ligand.sd && 
+        rbcavity -r new.prm -was > rbcavity.log && 

```

Comment:
Is the content of `rbcavity.log` interesting for the user? Here is completely lost. You may want to have a output log file or just remove the redirect.